### ISSUE TITLE
Events available on map but not in Calendar/Events list

### ISSUE THREAD

**Issue by tdutrion** [php-ug/php.ug#160]
Hi there!

Not sure how that works as I haven't read the code yet, but some of the events available on the map but not in the events list nor the calendar.

Examples are all the Scottish usergroups, that retrieve calendars from opentechcalendar (ical), and are not in the calendar page.

Any leads to fix that?

**Comment by heiglandreas** [comment]
This is indeed strange behaviour.... They should be listed in the calendar as well! I'll dig into that later today!

**Comment by heiglandreas** [comment]
Found the reason for it. The scottish usergroups (and some of the english ones) use opentechcalendar.co.uk which creates faulty iCalendar-files :frowning: 

To be sure that the calendar is ok I'm using sabre/vobjects validate-function which checks the calendars for errors. Those calendars are missing a ```DTSTAMP```-Field. As that's reported as error I've skipped those calendars (even though they are OK for the usage on php.ug). 

I'm fixing that currently and will also get in touch with the people from opentechcalendar.co.uk to give them the possibility to fix that.

Thanks for pointing out the issue!
